src/modules/85-sto/components/PipelineSteps/SnykStep/SnykStep.tsx

```diff
@@ -62,7 +62,7 @@ export class SnykStep extends PipelineStep<SnykStepData> {
       },
       auth: {
         domain: '',
-        accessToken: '<+secrets.getValue(“<your_snyk_token_secret>“)>',
+        accessToken: '<+secrets.getValue("<your_snyk_token_secret>")>',
```

Comment:
I think we should remove the extra < & >

```suggestion
        accessToken: '<+secrets.getValue("your_snyk_token_secret")>',
```